composer.json

```diff
@@ -17,29 +17,22 @@
         "github-domains": ["github.com"]
     },
     "require": {
-        "php": ">=5.6.0",
-        "symfony/framework-bundle": "~2.7,>=2.7|~3.0",
```

Comment:
would changing the versions for the symfony-sub-bundles to "^2.7|^3.0|^4.0" not work? It would be nicer to not require the complete symfony/symfony.